packages/sui-decorators/src/decorators/tracer/index.js

```diff
@@ -1,13 +1,11 @@
-import {performance} from 'perf_hooks'
+import perfNow from 'performance-now'
```

Comment:
This package uses native browser implementation for client-side code and a polyfilled version for Node-based code.